deepsea.spec.in

```diff
@@ -804,6 +804,7 @@ Group:          System/Console
 BuildRequires:  python3-setuptools
 Requires:       python3-setuptools
 Requires:       python3-click
+Requires:       deepsea
```

Comment:
Not sure if this is a good idea in this form, since it allows `deepsea` and `deepsea-cli` to coexist with different versions.

How about:

```
Requires:       deepsea = %{version}-%{release}
```

?

Reply:
Actually, I'm surprised this wasn't already there...

Reply:
Well spotted, thanks. Updated.